tests/Fixtures/app/config/config.yml

```diff
@@ -0,0 +1,33 @@
+framework:
+    secret:                           'PostmanGeneratorBundle'
+    router:
+        resource:                     '%kernel.root_dir%/config/routing.yml'
+        strict_requirements:          '%kernel.debug%'
+    validation:
+        enable_annotations:            true
+    serializer:
+        enable_annotations:            true
+    trusted_proxies:                   ~
+    test:                              ~
+    session:
+        storage_id:                    'session.storage.mock_file'
+
+doctrine:
+    dbal:
+        driver:                        "pdo_sqlite"
+        path:                          "%kernel.cache_dir%/db.sqlite"
+        charset:                       "UTF8"
+
+    orm:
+        auto_generate_proxy_classes:   "%kernel.debug%"
+        auto_mapping:                  true
+
+dunglas_api:
```

Comment:
We can probably support both v1 and v2 in the same version (like I've done in api-platform/schema-generator).